src/isolation/landlock.rs

```diff
@@ -294,17 +296,17 @@ pub(crate) fn initialize(
 	let mut uhyve_rw_paths: Vec<PathBuf> = vec![PathBuf::from("/dev/kvm")];
 	let mut uhyve_ro_dirs = Vec::new();
 
-	host_paths.iter().for_each(|host_path| {
+	for host_path in host_paths {
```

Comment:
Out of genuine curiosity, how come you changed the `for` loop here?

Reply:
This is a style choice (`for_each` is implemented as a `for` loop), but in this case, `.for_each` also has no benefits because the iterator chain in front of it was incredibly short.

In this case, it changes a few other things, too:
* passes an iterator which only borrows the input (where we previously had an allocation in `get_all_host_paths`)
* `.iter_mut()` is unnecessary here anyways (`.into_iter()` or `.iter()` suffices)